### DISCUSSION TITLE
Buying products from more than one seller at the same time

### DISCUSSION THREAD

**Discussion by exploitedtaken** [inspireui/support#5089] [Working on Features]
Hello, being able to buy products from more than one seller at the same time is confusing. For example, 5 products were purchased from 3 different vendors. On the order tracking page, there is no detail about which seller it was purchased from. Consequently, the customer does not know which seller to contact. An order should only have the option to buy products from one seller. It can be turned off and on upon request. There is no such option right now.

For example, products were purchased from 3 different vendors. The customer just wants to cancel one. Since there is no such option, the order is completely canceled.

This can be solved by requiring the availability of products from a single seller in one order.

It should give a warning when trying to add products from different vendors to the basket. For example, it may say that you must either reset your cart or you can buy this product after completing your order.

In the picture below, there is this option in the wcfm plugin but it is not added in the application.

Yours truly
Thank you

![fa5aae69-b5e1-480e-b2b9-8b48d1fbff6a](https://user-images.githubusercontent.com/13377870/112054217-5c83d200-8b66-11eb-945a-b4c283021994.png)

**Comment by CeoAlexMartinez** [comment]
Tienes razón, agreguen esta función

**Comment by Haniel5** [comment]
This feature is available "An order should only have the option to buy products from one seller"
